Dockerfile

```diff
@@ -262,4 +262,4 @@ USER mastodon
 # Expose default Puma ports
 EXPOSE 3000
 # Set container tini as default entry point
-ENTRYPOINT ["/usr/bin/tini", "--"]
```

Comment:
Accidental newline removal here?

Reply:
My editor added a newline, which I think is the correct behaviour here.